### PR TITLE
Improvement-spike-in-the-file-writes

### PR DESCRIPTION
**Pull request by VatsalJagani**

### CODE REVIEW

File: docs/user_guide/user_guide.md [hunk 1]
```diff
@@ -39,8 +39,10 @@ A few of the included alerts and reports are dependent on other reports being en
     * Palo Alto Firewall - Network Compromise - Inbound Traffic from Blocked IPs
     * Palo Alto Firewall - Network Compromise - Outbound Traffic to Blocked IPs
         * Dynamically Update Blocked IPs with HoneyDB
-    * Ransomware - Spike in File Writes 
+    * Ransomware - Spike in File Writes
         * Ransomware - Calculate UpperBound for Spike in File Writes
+    * Network Compromise - DDoS Behavior Detected
```
Comment (mahirchavda) [inline docs/user_guide/user_guide.md:44]: We need to remove this from this branch.

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 2]
```diff
@@ -221,7 +220,7 @@ search = | tstats `cs_summariesonly_endpoint` count min(_time) as firstTime max(
 | where count>=`cs_ransomware_min_file_writes` \
 | lookup cs_ransomware_file_writes_upperbound.csv dest OUTPUT upperBound, avg, stdev | where count>upperBound \
 | eval avg=round(avg, 2), stdev=round(stdev, 2), upperBound=round(upperBound, 2) \
-| eval cyences_severity = case(count>1000000, "critical", count>100000, "high", true(), "medium") \
+| eval eval cyences_severity = case(count>upperBound*3 AND count>100000, "critical", count>upperBound*2 AND count>10000, "high", true(), "medium") \
```
Comment (mahirchavda) [inline cyences_app_for_splunk/default/savedsearches.conf:223]: two evals.

Comment (VatsalJagani) [reply cyences_app_for_splunk/default/savedsearches.conf:223]: It just one right?

File: docs/user_guide/user_guide.md (hunk 1 above)
Comment (VatsalJagani) [reply docs/user_guide/user_guide.md]: Done

File: cyences_app_for_splunk/default/savedsearches.conf (hunk 2 above)
Comment (mahirchavda) [reply cyences_app_for_splunk/default/savedsearches.conf:223]: | eval eval

File: docs/user_guide/alerts_dashboards.md [hunk 3]
```diff
@@ -222,6 +222,11 @@ The Lansweeper dashboard also provides information about whether the IT asset is
     * Windows Devices 
 
 ## Network Devices
+* General alerts for all Network Data:
+    * Network Compromise - Basic Scanning
+    * Network Compromise - Inbound Vulnerable Traffic
+    * Network Compromise - DDoS Behavior Detected
```
Comment (mahirchavda) [inline docs/user_guide/alerts_dashboards.md:228]: Same here

File: docs/troubleshooting/troubleshooting.md [hunk 4]
```diff
@@ -130,3 +130,16 @@ Refer to the **App Installation and Configuration > Device Inventory > Backfill
...
+
+If you see below warning:
+    ```
+    'list' command: Limit of '100' for values reached. Additional values may have been truncated or ignored.
+    ```
+
+From any of the below alerts, then kindly `ignore` it, as that is expected behavior.
+    * Ransomware - Spike in File Writes
+    * Ransomware - Common Ransomware File Extensions
+    * Network Compromise - DDoS Behavior Detected
```
Comment (mahirchavda) [inline docs/troubleshooting/troubleshooting.md:145]: here as well

File: docs/user_guide/alerts_dashboards.md (hunk 3 above)
Comment (VatsalJagani) [reply docs/user_guide/alerts_dashboards.md:228]: Fixed.

File: docs/troubleshooting/troubleshooting.md (hunk 4 above)
Comment (VatsalJagani) [reply docs/troubleshooting/troubleshooting.md:145]: Fixed.

File: cyences_app_for_splunk/default/savedsearches.conf (hunk 2 above)
Comment (VatsalJagani) [reply cyences_app_for_splunk/default/savedsearches.conf]: Fixed.

**Pull request merged**